tool/generate-assets.sh

```diff
@@ -7,6 +7,28 @@ color="#f37736"
 rm -rf /tmp/nextcloud-neon
 mkdir -p /tmp/nextcloud-neon
 
+function parse_icons_css() {
+    icons_css_url="https://raw.githubusercontent.com/nextcloud/server/master/dist/icons.css"
```

Comment:
This file should be read from the git submodule at external/nextcloud-server instead

Reply:
How would I do that?